TF_Settings_Web/.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:unicorn/recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "settings": {
+        "react": {
+            "version": "detect"
+        }
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": ["react", "@typescript-eslint"],
+    "ignorePatterns": ["src/TouchFree"],
+    "rules": {
+        "indent": "off",
+        "linebreak-style": ["error", "windows"],
+        "quotes": ["error", "single"],
+        "semi": ["error", "always"],
+        "react/react-in-jsx-scope": "off",
+        "react/jsx-filename-extension": [1, { "extensions": [".ts", ".tsx"] }],
+        "@typescript-eslint/ban-types": [
+            "error",
+            {
+                "types": {
+                    "{}": false
+                },
+                "extendDefaults": true
+            }
+        ],
+        "@typescript-eslint/no-empty-interface": "warn",
+        "unicorn/prevent-abbreviations": "off",
+        "unicorn/filename-case": "off",
+        "unicorn/numeric-separators-style": "off",
+        "unicorn/no-null": "off",
+        "unicorn/no-zero-fractions": "off",
+        "unicorn/no-array-push-push": "off",
+        "unicorn/no-nested-ternary": "off"
```

Comment:
I think we should maybe increase the max line length from the default as by default it is 80 characters which is a bit short!
e.g.
`"max-len": 120`